test/request/querystring.js

```diff
@@ -33,8 +32,9 @@ describe('ctx.querystring=', function(){
     ctx.querystring = 'page=2&color=blue';
     ctx.url.should.equal('/store/shoes?page=2&color=blue');
     ctx.search.should.equal('?page=2&color=blue');
-    should(ctx.query).have.property('page', '2');
-    should(ctx.query).have.property('color', 'blue');
+    Object.keys(ctx.query).should.eql(['page', 'color']);
```

Comment:
Why this assert? The next ones will throw an error if the keys don't exist anyway.